process/block/preprocess/transactions.go

```diff
@@ -1103,6 +1128,8 @@ func (txs *transactions) ProcessMiniBlock(miniBlock *block.MiniBlock, haveTime f
 			return processedTxHashes, err
 		}
 
+		txs.saveAccountBalanceForAddress(miniBlockTxs[index].GetRcvAddr())
```

Comment:
why saving this here?

Reply:
Actually here is the point where cross txs dest me would be executed by the proposer when it creates a block. I need to take the initial balance of the receiver address before the tx is executed

Reply:
ok